bobtemplates/plone/content_type.py

```diff
@@ -26,6 +26,11 @@ def check_dexterity_type_name(configurator, question, answer):
     return answer
 
 
+def check_global_allow(configurator, answer):
+    if configurator.variables['dexterity_type_global_allow']:
+        raise SkipQuestion(u'global_allow is false, so we skip container name question.')
```

Comment:
this raises a flake8 error because the line is to long, 
please test such things by using tox upfront to check coding conventions are meet